sdk/src/invariant.ts

```diff
@@ -566,4 +566,19 @@ export class Invariant {
       block
     ) as Promise<SwapRouteTxResult>
   }
+  async getTickmap(
+    account: IKeyringPair,
+    poolKey: PoolKey,
+    currentTickIndex: bigint
+  ): Promise<bigint[][]> {
+    const result = (await sendQuery(
```

Comment:
That's not sufficient.
1. Create entire tickmap (create tickmap with all 0, then update initalized tick)
2. You can test u64 type here (will be better than bigint)